encrypted-media/clearkey-check-hdcp-policy.https.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <title>Encrypted Media Extensions: Check HDCP policy with Clear Key</title>
```

Comment:
Here and below and in the file names, to be accurate, this should be something like "Check status for minHdcpVersion policy"

Reply:
Done. Kept the files names simple with check-hdcp-version.